helm/default-apps-gcp/values.yaml

```diff
@@ -5,8 +5,17 @@ userConfig:
   cilium:
     configMap:
       values: |
+        ipam:
+          mode: kubernetes
```

Comment:
Be sure controller manager flags are set properly. https://github.com/giantswarm/giantnetes-terraform/blob/master/templates/files/manifests/controller-manager.yaml#L28

Reply:
Doing it here https://github.com/giantswarm/cluster-gcp/pull/141